### ISSUE TITLE
Using extract with multiple labels

### ISSUE THREAD

**Issue by mattyd100** [timcera/swmmtoolbox#9]
Can you provide me with an example of calling extract with multiple labels? I can get one set of labels to work, but not multiple ones, even though I have tried a variety of different formats.

These for work:
df = swmmtoolbox.extract(swmm_output_file_path, 'node,node_0,Total_inflow')
df = swmmtoolbox.extract(swmm_output_file_path, ['node,node_0,Total_inflow'])

But these don't:
df = swmmtoolbox.extract(swmm_output_file_path, 'node,node_0,Total_inflow node,node_1,Total_inflow')
df = swmmtoolbox.extract(swmm_output_file_path, [['node,node_0,Total_inflow'], ['node,node_1,Total_inflow']])

**Comment by timcera** [comment]
A single list of strings should work...

df = swmmtoolbox.extract(swmm_output_file_path, ['node,node_0,Total_inflow', 'node,node_1,Total_inflow'])

**Comment by mattyd100** [comment]
When I try that I get this error:

"The list ['node,node_0,Total_inflow', 'node,node_1,Total_inflow'] for "" should have 3 members according to function requirements."

**Comment by timcera** [comment]
I will take a look cause that should have worked.

Not ideal, but as a work-around you could so something like:

```
df0 = swmmtoolbox.extract(swmm_output_file_path, 'node,node_0,Total_inflow')
df1 = swmmtoolbox.extract(swmm_output_file_path, 'node,node_1,Total_inflow')
df = df0.join(df1)
```

**Comment by timcera** [comment]
Fixed and published as version 3.3.4.

Thanks!
